src/components-examples/angular/menu/menu-grouping/menu-grouping-example.html

```diff
@@ -0,0 +1,10 @@
+<button [sbbContextmenuTriggerFor]="menu" aria-label="Show stations" type="button"></button>
+<sbb-menu #menu="sbbMenu">
+  <label class="sbb-label">Railway stations in Bern</label>
```

Comment:
Maybe just use a span here? I'm uncertain about accessibility implications when using a label.

Reply:
I'd also suggest no to use label here. maybe in a span or div we should provide a proper aria role. I found `doc-subtitle` but I'm not sure if this would be appropriate.